instrumentation/active_job/lib/opentelemetry/instrumentation/active_job.rb

```diff
@@ -11,6 +11,39 @@ module OpenTelemetry
   module Instrumentation
     # Contains the OpenTelemetry instrumentation for the ActiveJob gem
     module ActiveJob
```

Comment:
Yes!

I was of thinking of making a generic "ingress" span Context that would be instrumentation agnostic that way any child span is able to access the ingress span and enrich it.

So instead of saying `Rack.current_span` or `ActiveJob.current_span` it would be `Ingress.current_span`.